Cktgen/runit-comparator.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 
 
-(cd Intel; python generate_comparator_v2.py)
+(cd Intel; python3 generate_comparator_v2.py)
 
 docker run --mount source=comparatorInputVol,target=/INPUT --name sam ubuntu
 docker cp Intel/__json sam:INPUT
 docker cp Intel/__json_grs sam:INPUT
 docker rm sam
 
-docker build -t cktgen .
+docker build --build-arg http_proxy=$http_proxy --build-arg https_proxy=$https_proxy -t cktgen .
```

Comment:
I don't need to do this (except for maybe the first time). The proxy spec shouldn't be in this script, at least not all the time. Maybe this script doesn't belong here either (in the Intel directory.)

Reply:
Good point. Do we want to move Cktgen to an internal repo - part of the major repo org?